omero/users/whatsnew.txt

```diff
@@ -1,12 +1,13 @@
 What's new for OMERO 5.1
 ========================
 
-From a user's perspective, OMERO 5.1 introduces units and some other
-stuff that others **will** write about.
-
-- Moving and deleting data is now faster.
-
-- There is **going to be** a new :help:`Getting Started guide
-  <getting-started-5.html>` for OMERO 5.1.
-
-.. note:: :doc:`demo-server` **is now available for OMERO 5.1.**
+From a user's perspective, OMERO 5.1 introduces support for units and new more
+flexible user-added metadata. Other updates and new features include:
+
+- improved performance for moving and deleting data
+- the ability to copy and paste rendering settings and select between
+  different users' settings in the UI
+- improved functionality of the OMERO ImageJ plugin allowing you to save ROIs
+  and overlays from ImageJ into OMERO
+- the ability to 'import to' specified groups, projects and datasets using
+  OMERO.dropbox or the Command Line Importer 
```

Comment:
At the moment all DropBox imports are to orphaned images and default group. No choice.

The CLI can be used to import to a specific group and target container, but in the case of a container it must already exist and its id must be known.

The intention in 5.1 is to allow that selection of group and container to be (optionally) automatic based on the source path of the original files. Here the containers would be created on demand if they didn't already exist. So, I guess the line needs re-wording a little, something like:

the ability to automatically 'import to' specified groups, projects and datasets from
OMERO.dropbox or the Command Line Importer using information from the file path

but worded a bit better?